Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -25,12 +25,15 @@
     <DefineConstants>$(DefineConstants);</DefineConstants>
     <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=868961</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
is this change really necessary for this PR? I think it's just a warning and not an error, right?

Reply:
I got an error, this is the actual message I got in the build (sorry for the delay by the way):
error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.

Reply:
was this error really preventing the build to succeed?

Reply:
It appears so:
2019-10-17T23:12:56.9871907Z   C:\Program Files\dotnet\sdk\3.0.100\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(198,5): error : Enabling license acceptance requires a license or a licenseUrl to be specified. The licenseUrl will be deprecated, consider using the license metadata. [D:\a\DotNetEssentials\DotNetEssentials\Xamarin.Essentials\Xamarin.Essentials.csproj]
2019-10-17T23:12:56.9872757Z     16 Warning(s)
2019-10-17T23:12:56.9873018Z     1 Error(s)
2019-10-17T23:12:56.9873376Z Time Elapsed 00:00:17.11
2019-10-17T23:12:57.1364769Z ##[error]Process completed with exit code 1.